packages/react-mutation-mapper/src/util/OncoKbUtils.ts

```diff
@@ -411,141 +267,14 @@ export function mergeAlterations(alterations: string | string[]) {
-                            evidence
-                        );
-                        _treatment.level = evidence.levelOfEvidence;
-                        _treatment.content = evidence.treatments;
-                        _treatment.description = description || '';
-
-                        if (
-                            LEVELS.sensitivity.indexOf(
-                                getLevel(evidence.levelOfEvidence)
-                            ) !== -1
-                        ) {
-                            sensitivityTreatments.push(_treatment);
-                        } else {
-                            resistanceTreatments.push(_treatment);
-                        }
-                    }
-                }
-            });
-
-            if (datum.alteration.length > 0) {
-                datum.mutationEffect = datum.alteration[0];
-            }
-
-            datum.treatments.sensitivity = sensitivityTreatments;
-            datum.treatments.resistance = resistanceTreatments;
-            // id.split('*ONCOKB*').forEach(function(_id) {
-            //     result[_id] = datum;
-            // })
-            result[id] = datum;
-        });
-    }
-
-    return result;
-}
-
-export function initEvidence() {
-    return {
-        id: '',
-        gene: {},
-        alteration: [],
-        prevalence: [],
-        progImp: [],
-        treatments: {
-            sensitivity: [],
-            resistance: [],
-        }, //separated by level type
-        trials: [],
-        oncogenic: '',
-        oncogenicRefs: [],
-        mutationEffect: {},
-        mutationEffectRefs: [],
-        summary: '',
-    };
-}
-
-export function getEvidenceQuery(
-    mutation: Mutation,
-    resolveEntrezGeneId: (mutation: Mutation) => number,
-    resolveTumorType: (mutation: Mutation) => string
-) {
-    return generateQueryVariant(
-        resolveEntrezGeneId(mutation),
-        resolveTumorType(mutation),
-        mutation.proteinChange,
-        mutation.mutationType,
-        mutation.proteinPosStart,
-        mutation.proteinPosEnd
-    );
-}
-
-export function getTumorTypeFromEvidence(evidence: any) {
-    var tumorType = _.isObject(evidence.tumorType)
-        ? evidence.tumorType.name
-        : evidence.subtype || evidence.cancerType;
-    var oncoTreeTumorType = '';
-
-    if (_.isObject(evidence.oncoTreeType)) {
-        oncoTreeTumorType = evidence.oncoTreeType.name
-            ? evidence.oncoTreeType.name
-            : evidence.oncoTreeType.mainType
-            ? evidence.oncoTreeType.mainType.name
-            : '';
-    }
-
-    if (oncoTreeTumorType) {
-        tumorType = oncoTreeTumorType;
-    }
-
-    return tumorType;
+export function getTumorTypeName(tumorType: TumorType) {
+    return tumorType
+        ? tumorType.name
+            ? tumorType.name
+            : tumorType.mainType
+            ? tumorType.mainType.name
+            : ''
+        : '';
```

Comment:
nested ternary operators might be confusing, should we use `if/else` blocks instead?

Reply:
I was going to suggest doing something with `&&` and `||` but I think its too messy, agree with @onursumer